### PR TITLE
Us/533/visibility options

### PR DESCRIPTION
**Pull request by juanfran**

### CODE REVIEW

File: app/coffee/modules/kanban/main.coffee [hunk 1]
```diff
@@ -869,6 +876,7 @@ CardDataDirective = ($template, $translate, avatarService, projectService, dueDa
             initializeZoom = false
 
             onChange = () =>
+                console.log $scope.vm
```
Comment (Xaviju) [inline app/coffee/modules/kanban/main.coffee:879]: `$scope.vm`

File: app/coffee/modules/taskboard/main.coffee [hunk 2]
```diff
@@ -826,13 +826,14 @@ TaskboardSquishColumnDirective = (rs) ->
...
             column = $el.find(".squish-status-#{statusId}")
 
-            if width
-                column.css('max-width', width)
-            else
-                if $scope.ctrl.zoomLevel == '0'
-                    column.css("max-width", 148)
-                else
-                    column.css("max-width", maxColumnWidth)
+            # TODO in #701
```
Comment (Xaviju) [inline app/coffee/modules/taskboard/main.coffee:829]: I guess this is intended

File: app/modules/components/card/card-templates/card-assigned-to.jade [hunk 3]
```diff
@@ -31,6 +32,20 @@
...
+    .card-user-avatar
+        img(
+            src!="<%- avatars[vm.item.getIn(['assigned_to', 'id'])].url %>"
+            title!="<%- avatars[vm.item.getIn(['assigned_to', 'id'])].fullName %>"
+            alt!="<%- avatars[vm.item.getIn(['assigned_to', 'id'])].fullName %>"
+            style!="background-color: <%- avatars[vm.item.getIn(['assigned_to', 'id'])].bg || '' %>"
+        )
+        <% if (vm.item.getIn(['model', 'is_iocaine'])) { %>
+        .card-iocaine-user-bg
+            svg(xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 28 17")
```
Comment (Xaviju) [inline app/modules/components/card/card-templates/card-assigned-to.jade:45]: I guess this intended and this SVG should not be sprite

File: app/coffee/modules/kanban/main.coffee (hunk 1 above)
Comment (juanfran) [reply app/coffee/modules/kanban/main.coffee]: fixed

File: app/coffee/modules/taskboard/main.coffee (hunk 2 above)
Comment (juanfran) [reply app/coffee/modules/taskboard/main.coffee:829]: intended

File: app/modules/components/card/card-templates/card-assigned-to.jade (hunk 3 above)
Comment (juanfran) [reply app/modules/components/card/card-templates/card-assigned-to.jade:45]: intended, very specific use case. i'll make the sprite heavier for an small use case

**Pull request merged**